core/server/master/src/test/java/alluxio/master/meta/AlluxioMasterRestServiceHandlerTest.java

```diff
@@ -431,4 +432,19 @@ public void getWorkerInfoList() {
       response.close();
     }
   }
+
+  @Test
+  public void isMounted() throws Exception {
+    String s3Uri = "s3a://test/dir_1/dir-2";
+    String hdfsUri = "hdfs://test";
+
+    Map<String, MountPointInfo> mountTable = new HashMap<>();
+    mountTable.put("/s3", new MountPointInfo().setUfsUri(s3Uri));
+    when(mFileSystemMaster.getMountTable()).thenReturn(mountTable);
+
+    assertFalse(mHandler.isMounted(s3Uri));
+    assertTrue(mHandler.isMounted(MetricsSystem.escape(new AlluxioURI(s3Uri))));
```

Comment:
can you also check if this succeeds for s3Uri and hdfsUri ending with /. is mountTable key guaranteed to not end with /?

Reply:
the ufs uri in mount table  is guaranteed not to end with `/`

Reply:
to be safer, I added logic to trim the trailing slash